ee/tabby-db/migrations/07-email-service-credentials/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE email_service_credentials(
```

Comment:
```suggestion
CREATE TABLE email_service_credential(
```

sorry, should be singular